sync/class.jetpack-sync-actions.php

```diff
@@ -448,9 +448,32 @@ static function get_sync_status() {
 		$cron_timestamps = array_keys( _get_cron_array() );
 		$next_cron       = $cron_timestamps[0] - time();
 
+		$checksums = array();
+
+		if ( ! is_null( $fields ) ) {
```

Comment:
Is `is_null` the best here, since in some scenarios `$fields` may be empty array:
https://github.com/Automattic/jetpack/pull/12209/files#diff-7eba8bca1502098c08e8818da6353b18R56

Should we use `empty` instead? It should cover both empty array and `null`:

```suggestion
		if ( ! empty( $fields ) ) {
```

It should avoid warnings like this:

```
PHP Warning:  in_array() expects parameter 2 to be array, null given in /var/www/html/wp-content/plugins/jetpack/sync/class.jetpack-sync-actions.php on line 469
```